RFS/src/test/java/org/opensearch/migrations/bulkload/common/DocumentReindexerTest.java

```diff
@@ -31,7 +33,7 @@
 class DocumentReindexerTest {
 
     private static final int MAX_DOCS_PER_BULK = 3;
-    private static final int MAX_BULK_SIZE = 1000;
+    private static final int MAX_BYTES_PER_BULK_REQUEST = 1024*1024;
```

Comment:
Is this size pre or post-transformations?  Do you have a test that verifies those limits when transformations are being used?

Reply:
The size here is post-transformations

Reply:
added reindex_shouldBufferByTransformedSize to verify this